cypress/integration/SearchPage.feature

```diff
@@ -37,14 +37,6 @@ Feature: User interacts with Search page
     #     And you click search
     #     Then you should see no results found
 
-    Scenario: Searching using attributes in Tracker Program throws error
-        Given you are on the default search page
-        When you select the search domain Malaria Case diagnosis
-        And you expand the attributes search area
-        And you fill in the first name with values that will return an error
```

Comment:
With the filter value escaped, this scenario is obsolete. Please let me know if you know of another way to make the API request fail and I can adapt the test instead of removing it.